### PR TITLE
[PR to PR] NRI raider de-conflicting

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Should be merged into #17920 
Does what it says on the tin. That is, de-conflicting stuff as best as I could.
Should also be noted that I've removed a good chunk of some revolver-related code due to it conflicting with the PR and me not understanding how on Earth would I even fix it.

## How This Contributes To The Skyrat Roleplay Experience
Codebase will compile you know.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
I am literally localhosting this and it all works the way it should. While you might think that this screenshot shows nothing it actually shows you that the codebase compiles.

image

Oh and NRI raiders are spawn correctly and actually work yeah.
image
image

</details>

## Changelog
Not important.

### CODE REVIEW

**Comment by Stalkeros2** [comment]
> You currently have a negative Fix/Feature pull request delta of -20. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

image

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 1]
```diff
@@ -130,52 +130,6 @@
 		"Black Panther" = "c38_panther"
 	)
 
-	/// Used to avoid some redundancy on a revolver loaded with 357 regarding misfiring while being wrenched.
```
Comment (Zonespace27) [inline code/modules/projectiles/guns/ballistic/revolver.dm:133]: don't remove tg code

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/ballistic/revolver.dm (hunk 1 above)
Comment (Stalkeros2) [reply code/modules/projectiles/guns/ballistic/revolver.dm:133]: I'm fairly certain that I've quite literally just reverted it entirely to /tg/ code.

Comment (Stalkeros2) [reply code/modules/projectiles/guns/ballistic/revolver.dm:133]: image
I am not wrong. This screenshot is upstream.

File: modular_skyrat/modules/encounters/code/nri_raiders.dm [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+	var/number = rand(1,99)
+	///Station name one is the most important pick and is pretty much the station's main argument against getting fined, thus it better be mostly always right.
+	var/station_designation = pick_weight(list(
+	"Nanotrasen Research Station" = 70,
+	"Nanotrasen Refueling Outpost" = 5,
+	"Interdyne Pharmaceuticals Chemical Factory" = 5,
+	"Free Teshari League Engineering Station" = 5,
+	"Agurkrral Military Base" = 5,
+	"Sol Federation Embassy" = 5,
+	"Novaya Rossiyskaya Imperiya Civilian Port" = 5,
```
Comment (Fikou) [inline modular_skyrat/modules/encounters/code/nri_raiders.dm]:
```suggestion
		"Nanotrasen Research Station" = 70,
		"Nanotrasen Refueling Outpost" = 5,
		"Interdyne Pharmaceuticals Chemical Factory" = 5,
		"Free Teshari League Engineering Station" = 5,
		"Agurkrral Military Base" = 5,
		"Sol Federation Embassy" = 5,
		"Novaya Rossiyskaya Imperiya Civilian Port" = 5,
```

File: modular_skyrat/modules/encounters/code/nri_raiders.dm [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+	"unethical hiring practices and unfair payment allocation for the NRI citizens",
+	"recently discovered BSA-[number] or similar model in close proximity to the neutral space aboard this or nearby affiliated facility",
+	)
+	///"wrong" = Loosely based accusations that can be easily disproven if people think.
+	var/wrong_pick = pick(
+	"inadequate support of the local producer",
+	"unregulated production of Gauss weaponry aboard this installation",
+	"SolFed-backed stationary military formation on the surface of Indecipheres",
+	"AUTOMATED REGULATORY VIOLATION DETECTION SYSTEM CRITICAL FAILURE. PLEASE CONTACT AND INFORM THE DISPATCHED AUTHORITIES TO RESOLVE THE ISSUE. \
+		ANY POSSIBLE INDENTURE HAS BEEN CLEARED. WE APOLOGIZE FOR THE INCONVENIENCE",
```
Comment (Fikou) [inline modular_skyrat/modules/encounters/code/nri_raiders.dm]:
```suggestion
		"inadequate support of the local producer",
		"unregulated production of Gauss weaponry aboard this installation",
		"SolFed-backed stationary military formation on the surface of Indecipheres",
		"AUTOMATED REGULATORY VIOLATION DETECTION SYSTEM CRITICAL FAILURE. PLEASE CONTACT AND INFORM THE DISPATCHED AUTHORITIES TO RESOLVE THE ISSUE. \
		ANY POSSIBLE INDENTURE HAS BEEN CLEARED. WE APOLOGIZE FOR THE INCONVENIENCE",
```

File: code/modules/projectiles/guns/ballistic/revolver.dm (hunk 1 above)
Comment (RimiNosha) [reply code/modules/projectiles/guns/ballistic/revolver.dm:133]: Hmmm, yes, undocumented nonmodular edits.

Comment (Zonespace27) [reply code/modules/projectiles/guns/ballistic/revolver.dm:133]: *incredibly* based

**Review by Zonespace27 — APPROVED** [review]

**Pull request merged**